### ISSUE TITLE
Plans for merging abstract models branch

### ISSUE THREAD

**Issue by blablacio** [dakrauth/django-swingtime#23]
@dakrauth I'm wondering if you're planning to merge the abstract models branch anytime soon?

**Comment by dakrauth** [comment]
Yes, I hope to release that version within the next week or two.

**Comment by blablacio** [comment]
@dakrauth I'm willing to help out speed the release up, let me know how I can do that.
